poi-scratchpad/src/main/java/org/apache/poi/hslf/usermodel/HSLFTextRun.java

```diff
@@ -440,6 +440,23 @@ public void setFontColor(PaintStyle color) {
         setFontColor(rgb);
     }
 
+
+    @Override
+    public PaintStyle getHighlightColor() {
+        return null;
+    }
+
+    @Override
+    public void setHighlightColor(final Color color) {
```

Comment:
why add these methods? - if this stuff is only supported on XSLFTextRun, then only that class should have these methods.

Reply:
I agree, but the common rendering in DrawTextParagraph does not have access to the XSLF* classes, so cannot be cast - I chose this as a tradeoff. If you have a better suggestion then Im happy to adjust as needed.